src/script_lua.c

```diff
@@ -56,10 +56,10 @@ static char *libraries_allow_list[] = {
 };
 
 /* Lua API globals */
-static char *redis_api_allow_list[] = {
+static char *server_api_allow_list[] = {
     SERVER_API_NAME,
     REDIS_API_NAME,
-    "__redis__err__handler", /* error handler for eval, currently located on globals.
+    "__server__err__handler", /* error handler for eval, currently located on globals.
```

Comment:
The existence of this global is undocumented, but I guess this change can break user scripts if they access this global, like calling it manually `EVAL "__redis__err__handler(foo)" 0` or something.

@madolson You have seen many strange things from users. Do you think this would be a risk?

Reply:
Yeah. I guess I would expect this list to include both the err handlers. I can't think of a reason a user would touch this, but a user could theoretically be manipulating this since it's in the accessible globals list.

Reply:
Alright, added the suggested change since it is simple change. 
I generally don't prefer to keep backwards compatibility for internal variables. That just makes it too complicated.

Reply:
It's not an internal variable though. It's accessible by users, though undocumented.

Both is safe, but we could also just have kept only the redis error handler here instead of adding one more. :-)